test/core.js

```diff
@@ -264,6 +264,14 @@ describe('snabbdom', function() {
       patch(vnode1, vnode2);
       assert.equal(elm.src, undefined);
     });
+    it('changes an elements value-prop it was changed in the dom', function() {
```

Comment:
```suggestion
    it('sets the `value` prop if it was changed in the DOM', function() {
```

Reply:
The name change is ok.

I hope it won't take too long to get into a release, because it's a quite annoying problem if you want any kind of input-field completion/formatting.

One possible extension would be enabling similar behaviour for the 'selected' and 'checke' props. But those aren't as often overwritten as value so it's not as important.

But we may need a documentation update for this.

Reply:
I would be in favor of doing this for `selected` and `checked` as well, because overwritting these does happen (at least in my application). Thus, it would be really convenient to solve these properties, too.